docs/branch_activation.md

```diff
@@ -0,0 +1,35 @@
+[← Back to README](../README.md)
+
+## Branch Activation Process
+
+To use e2e tests in other repositories, you must follow a branch activation process. This process is critical for ensuring that tests run smoothly and that branches remain active during development.
```

Comment:
minor typo, should be -> _To use e2e tests in other repositories, you must follow a branch activation process. This process is critical for ensuring that tests run smoothly, and that branches remain active during development._

Reply:
Done